packages/functions_client/lib/src/functions_client.dart

```diff
@@ -71,11 +71,13 @@ class FunctionsClient {
     String functionName, {
     Map<String, String>? headers,
     Map<String, dynamic>? body,
+    Map<String, String>? queryParameters,
```

Comment:
Should we align the naming here with the Swift library and call it `query` instead of `queryParameters`?

Reply:
I think we should keep the wording of the dart ecosystem, which uses `queryParameters` in the `Uri` class for a map and `query` for the string representation.

Reply:
Sure, fair enough.